settings.gradle.kts

```diff
@@ -35,6 +35,9 @@ plugins {
 
 includeBuild("hedera-platform")
 
+// If you turn this is on, 'platform-sdk' modules are built from source
+// includeBuild("platform-sdk")
```

Comment:
Adding this line will _include_ `platform-sdk` into the `hedera-node` build. Then all modules needed will be built from source automatically when you work on `hedera-node`. The `swirldsVersion` in `hedera-platform/build.gradle.kts` is ignored then.